src/osvr/PluginHost/RegistrationContext.cpp

```diff
@@ -77,12 +80,13 @@ namespace pluginhost {
                                         std::string const &name,
                                         OSVR_PluginRegContext ctx,
                                         bool shouldRethrow = false) {
-        OSVR_DEV_VERBOSE("Trying to load a plugin with the name " << name);
+        auto log = util::log::make_logger("PluginHost");
+        log->debug() << "Trying to load a plugin with the name " << name;
```

Comment:
It's something I've noted for future work. It should be possible, yes.

Currently spdlog doesn't support this. You can have different levels for different loggers (e.g., PluginHost versus Server), but not per-sink.